spec/models/order_spec.rb

```diff
@@ -1,6 +1,33 @@
 require "rails_helper"
 
 RSpec.describe Order, type: :model do
+  context "valid attributes" do
+
+  let!(:loan_request) { LoanRequest.create(title: "Farm Tools",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Use 2 (not 0) spaces for indentation.<br>Block body expression is on the same line as the block start.